scripts/globals/weaponskills.lua

```diff
@@ -1100,6 +1100,13 @@ xi.weaponskills.getHitRate = function(attacker, target, capHitRate, bonus, isSub
         bonus = 0
     end
 
+    if
+        calcParams ~= nil and
+        calcParams.bonusAcc ~= nil
+    then
+        bonus = bonus + calcParams.bonusAcc
```

Comment:
@TiberonKalkaz What about ranged weaponskills and getRangedHitRate ? I think gorgets should impact that as well?

Reply:
Good question - I had tested with Sidewinder but I'll check a multi hit ranged ws and adjust if needed

Reply:
Yup, good catch @TracentEden  - updated in latest version